### DISCUSSION TITLE
How to remove stuck or failed partitions?

### DISCUSSION THREAD

**Discussion by lydialimlh** [dagster-io/dagster#19044] [Q&A] [answered]
Hey guys this is a question related to an issue I am facing: https://github.com/dagster-io/dagster/issues/18904 

Is it possible to reset the state of partitions that have ran and failed? (i.e. click on the red lines representing failed partitions in the UI and tell Dagster to forget that we ever ran those partitions?)

**Comment by clairelin135** [comment, marked as answer]
Hi @lydialimlh. Unfortunately this is not possible at this time, Dagster does not support per-partition wiping. Here's an issue for tracking: https://github.com/dagster-io/dagster/issues/14749
